src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -246,7 +243,7 @@
     public void onStartup() {
         super.onStartup();
         /* check for successful import of gcnvkernel */
-        PythonScriptExecutor.checkPythonEnvironmentForPackage("gcnvkernel");
+        PythonScriptExecutor.checkPythonEnvironmentForPackage("numpy");
```

Comment:
Why this change?

Reply:
Oops, this was to get it to run in IDE. Thanks for catching this!